tests/unit/util/test_file_converter.py

```diff
@@ -193,3 +211,17 @@ def test_csv_no_index(tmp_output_path, netcdf_filepath):
             else:
                 assert dt.datetime.fromisoformat(row[0])
 
+
+def test_csv_reformat_bad_names_removed(tmp_output_path, csv_filepath):
+    """Test that dataframes that have been reformatted using 'csv_reformatter'
+    no longer contain columns of data we do not require."""
+    tmp_df = fc.csv_reformatter(csv_filepath)
+    assert "status" not in tmp_df.columns
+
+
+def test_csv_reformat_syntax_corrected(tmp_output_path, csv_filepath):
+    """Test that dataframes that have been reformatted using 'csv_reformatter'
+    no longer contain invalid syntax in headers."""
+    tmp_df = fc.csv_reformatter(csv_filepath)
+    assert "PM<sub>10</sub> particulate matter (Hourly measured)" \
```

Comment:
I'm curious what line length setting you're using, I'm using 120 characters, but this looks like you're using 80? It'd be good to use the same setting, so we don't end up accidentally reformatting each others code